pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -178,15 +206,19 @@ func newGCECloud(config io.Reader) (*GCECloud, error) {
 		if cfg.Global.TokenURL != "" {
 			tokenSource = newAltTokenSource(cfg.Global.TokenURL, cfg.Global.TokenBody)
 		}
+		if cfg.Global.Multizone != "" {
```

Comment:
Related to above comment, this config param needs to be a boolean. As it stands, unless I'm mistaken, setting MULTIZONE=false or MULTIZONE=0 will be equivalent to setting MULTIZONE=true, which is downright confusing.